google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/AppEngineFlexibleSupportProvider.java

```diff
@@ -162,8 +163,11 @@ private static boolean hasFlexibleDeploymentConfiguration(List<RunConfiguration>
         .map(runConfig -> ((DeployToServerRunConfiguration) runConfig).getDeploymentConfiguration())
         .filter(deployConfig -> deployConfig instanceof AppEngineDeploymentConfiguration)
         .anyMatch(
-            deployConfig ->
-                ((AppEngineDeploymentConfiguration) deployConfig).getEnvironment().isFlexible());
+            deployConfig -> {
+              AppEngineEnvironment environment =
+                  ((AppEngineDeploymentConfiguration) deployConfig).getEnvironment();
+              return environment != null && environment.isFlexible();
```

Comment:
So this is an expected case when environment could be `null`?

Reply:
I'm not 100% sure how it was null in the first place. My reasoning is that it is harmless to silently skip this case since, if you look at what this method is used for, it will just go ahead and create a working deployment run configuration. wdyt?